v5.14.6 (#38602)
mui/material-ui: CHANGELOG.md

# [Versions](https://mui.com/versions/)

## 5.14.6

<!-- generated comparing v5.14.5..master -->

_Aug 23, 2023_

A big thanks to the 21 contributors who made this release possible. Here are some highlights ✨:

- 🚀 Added [Accordion](https://mui.com/joy-ui/react-accordion/) component to Joy UI (#38164) @siriwatknp

> - 🎉 Added Material You Input and ButtonBase components to `material-next` (#38319) @DiegoAndai @mj12albert

Reviewer: ```suggestion
- 🎉 Added Material You InputBase and ButtonBase components to `material-next` (#38319) @DiegoAndai @mj12albert
```

It should be `InputBase` and not `Input` ~ (which is the standard variant Input)

Author: Done!

# [Versions](https://mui.com/versions/)

## 5.14.6
<!-- generated comparing v5.14.5..master -->
_Aug 23, 2023_

A big thanks to the 21 contributors who made this release possible. Here are some highlights ✨:

- 🚀 Added [Accordion](https://mui.com/joy-ui/react-accordion/) component to Joy UI (#38164) @siriwatknp

> - 🎉 Added Material You Input and ButtonBase components to `material-next` (#38319) @DiegoAndai @mj12albert

Author: cc @DiegoAndai @mj12albert if you have better words.

> - 🔋 Added zero-runtime support for [Linaria](https://linaria.dev/) (#38378) @brijeshb42

Author: @brijeshb42 is my understanding is correct?

Reviewer 2: It doesn't feel like we added support for it yet. IMHO, this sets wrong expectation for developers, for example, until there is documentation, we can't say a feature is available IMHO.

Maybe?

```suggestion
- 🔋 First iteration on a CSS-in-JS compatible with Server Components (#38378) @brijeshb42
```

Reviewer 3: ☝️ Maybe missing what "a CSS-in-JS" is? Here's an option:

> 🔋 First iteration on a CSS-in-JS _**engine**_ compatible with Server Components

# [Versions](https://mui.com/versions/)

## 5.14.6

<!-- generated comparing v5.14.5..master -->

_Aug 23, 2023_

A big thanks to the 21 contributors who made this release possible. Here are some highlights ✨:

> - 🚀 Added [Popup](https://mui.com/base-ui/react-popup/) component to Base UI (#37960) @michaldudak

Reviewer 2: It would be awesome to have a paragraph or two on this. For example, to share our plan when it comes to Popover vs. Popper vs. Popup. But also, when to use each, and lastly, why is there a new one. I personally didn’t understand this. Why the library it uses under the hood feels like an implementation details.

If I would try to rationalize it, it seems to be: since we are moving Popper to no longer use popper.js, we do no longer need to call this component Popper. We are renaming it to Popup, which keeping the API almost identical for easy migration.

> - 🚀 Added the [Popup](https://mui.com/base-ui/react-popup/) component to Base UI (#37960) @michaldudak

Reviewer 2: @michaldudak It would be great to have a paragraph or two on this. For example, to share our plan when it comes to Popover vs. Popper vs. Popup. But also, when to use each, and lastly, why is there a new one.

I personally didn't capture the need for it. If I would try to guess it, it seems to be: since we are moving Popper to no longer use popper.js, we do no longer need to call this component Popper. We are renaming it to Popup to distinguish both versions between Material UI and Base UI, which keeping the API almost identical for easy migration.

I guess the alternative was to do a PopperV2, like GridV2.

Reviewer 2: In https://github.com/mui/mui-x/issues/10006#issuecomment-1680523234 I felt that we could turn Popover to be a bit of a kitchen sink for creating popups, could support modal and non-modal modes, could be used by: select, autocomplete, menu, date pickers. And under the hood, would use Popup, Modal, and animations.

Reviewer 4: I added an explanation on both the Popup and Popper docs pages, so hopefully, this won't be confusing. Here we can add:

```suggestion
- 🚀 Added the [Popup](https://mui.com/base-ui/react-popup/) component to Base UI (#37960) @michaldudak 
  It's intended to replace the Popper component, which uses the deprecated Popper JS library. The Popup is built on top of Floating UI and has a similar API to the Popper.
```

- 🚀 Added the [Accordion](https://mui.com/joy-ui/react-accordion/) component to Joy UI (#38164) @siriwatknp

> - 🚀 Added the Material You InputBase and ButtonBase components to `material-next` (#38319) @DiegoAndai @mj12albert

Reviewer 5: ```suggestion
- 🚀 Added InputBase and ButtonBase components to `material-next` (#38319) @DiegoAndai @mj12albert
```

Lets remove the "Material You" as these base components do not implement Material You styles

> - 🔋 Added zero-runtime support for [Linaria](https://linaria.dev/) (#38378) @brijeshb42

Author: ```suggestion
- 🔋 First iteration on the CSS-in-JS engine compatible with Server Components (#38378) @brijeshb42
```

### `@mui/material@5.14.6`

- [Modal] Update it to use the useModal hook (#38498) @mnajdova
- [Select] Add `root` class to `SelectClasses` (#38424) @sai6855
- [Skeleton] Soften the pulse animation (#38506) @oliviertassinari
- [TextField] Fix onClick regressions handling changes (#38474) @mj12albert
- [TextField] Fix TextField onClick test (#38597) @mj12albert

### `@mui/base@5.0.0-beta.12`

- [Popup] New component (#37960) @michaldudak

### `@mui/joy@5.0.0-beta.3`

- [Accordion] Add Joy UI Accordion components (#38164) @siriwatknp
- Miscellaneous fixes (#38462) @siriwatknp

> - Fix typo: Classname -> Class name for consistency (#38510) @alexfauquette

Reviewer 2: ```suggestion
- [docs][joy-ui] Fix typo: Classname -> Class name for consistency (#38510) @alexfauquette
```

# [Versions](https://mui.com/versions/)

## 5.14.6

<!-- generated comparing v5.14.5..master -->

_Aug 23, 2023_

A big thanks to the 21 contributors who made this release possible. Here are some highlights ✨:

- 🚀 Added the [Popup](https://mui.com/base-ui/react-popup/) component to Base UI (#37960) @michaldudak
  It's intended to replace the Popper component, which uses the deprecated Popper JS library. The Popup is built on top of Floating UI and has a similar API to the Popper.
- 🚀 Added the [Accordion](https://mui.com/joy-ui/react-accordion/) component to Joy UI (#38164) @siriwatknp

> - 🚀 Added InputBase and ButtonBase components to `material-next` (#38319) @DiegoAndai @mj12albert

Reviewer 6: This was done for testing purposes, does it have any value for the consumers in order to be a highlight?

Author: To highlight the progress we have made.

> - 🔋 First iteration on the CSS-in-JS engine compatible with Server Components (#38378) @brijeshb42

Reviewer 6: ```suggestion
- 🔋 First iteration on the zero-runtime styling engine compatible with Server Components (#38378) @brijeshb42
```

I would phrase it this way, but no hard preference :)
